includes/stats-calculator.php

```diff
@@ -253,4 +253,41 @@ public function event_has_stats( int $event_id ): bool {
 
 		return ! empty( $stats->rows() );
 	}
+
+	public function unreview_translations( int $event_id ) {
```

Comment:
I assume this is a typo

```suggestion
	public function unreviewed_translations( int $event_id ) {
```

Reply:
I removed this while rebasing, its no longer needed.